source/features/quick-repo-deletion.tsx

```diff
@@ -65,6 +191,17 @@ void features.add(import.meta.url, {
 	init: initSettingsPage,
 });
 
+features.add(import.meta.url, {
+	asLongAs: [
+		pageDetect.isRepoSettings,
+		pageDetect.isForkedRepo,
+		canUserDeleteRepository,
+		isRepoUnpopular,
```

Comment:
Only leave repo settings. The request is very explicit so if they want to delete other repos in one ctrl-alt-click then let them do it.